From 03e5a4c9177a4f791cc7f3e99d038b4a0d7ba82c Mon Sep 17 00:00:00 2001
From: Murali Karicheri <m-karicheri2@ti.com>
Date: Mon, 11 Dec 2017 16:42:32 -0500
Subject: [PATCH 119/120] ARM: dts: am571x-idk: enable prp and ptp on ICSS1

This enables PRP and PTP on ICSS1 so that SAN PTP can be brought up
on ICSS1. The PRP firmware is used in EMAC mode to support SAN PTP
function.

Signed-off-by: Murali Karicheri <m-karicheri2@ti.com>
---
 arch/arm/boot/dts/am571x-idk.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/am571x-idk.dts b/arch/arm/boot/dts/am571x-idk.dts
index 793f99f..1a067b1 100644
--- a/arch/arm/boot/dts/am571x-idk.dts
+++ b/arch/arm/boot/dts/am571x-idk.dts
@@ -107,8 +107,8 @@
 		pruss1_emac0: ethernet-mii0 {
 			phy-handle = <&pruss1_eth0_phy>;
 			phy-mode = "mii";
-			interrupts = <20>, <22>;
-			interrupt-names = "rx", "tx";
+			interrupts = <20>, <22>, <23>;
+			interrupt-names = "rx", "tx", "ptp_tx";
 			/* Filled in by bootloader */
 			local-mac-address = [00 00 00 00 00 00];
 		};
@@ -116,8 +116,8 @@
 		pruss1_emac1: ethernet-mii1 {
 			phy-handle = <&pruss1_eth1_phy>;
 			phy-mode = "mii";
-			interrupts = <21>, <23>;
-			interrupt-names = "rx", "tx";
+			interrupts = <21>, <23>, <24>;
+			interrupt-names = "rx", "tx", "ptp_tx";
 			/* Filled in by bootloader */
 			local-mac-address = [00 00 00 00 00 00];
 		};
-- 
1.9.1

